test/cql-pytest/run.py

```diff
@@ -98,9 +132,13 @@ def restart_with_dir(old_pid, run_cmd_generator, dir):
 # of copying it to stdout.
 def abort_run_with_dir(pid, tmpdir):
     try:
-        os.killpg(pid, 9)
+        # We can use os.killpg(pid, signal.SIGKILL) to always kill Scylla
+        # immediately with SIGKILL, but trying SIGTERM first has a tiny
+        # overhead (0.02 seconds), and allows testing the shutdown path
+        # and flushing test-coverage data in a coverage run.
+        killpg_retry(pid, signal.SIGTERM, 10)
         os.waitpid(pid, 0) # don't leave an annoying zombie
```

Comment:
`killpg_retry()` already waited for the pid, so this can be removed, but it is not important.

Reply:
The SIGKILL killing fallback didn't (but probably should to avoid _this_ one)

Reply:
Exactly, I did this deliberately - I wanted my killpg variant to not guarantee more than the original killpg did. So the waiting for the zombie after the killpg remained where it was before the patch. Yes, I could have moved it inside killpg_retry().